pyuntl/metadata_generator.py

```diff
@@ -174,7 +174,7 @@ def highwiredict2xmlstring(highwire_elements, ordering=HIGHWIRE_ORDER):
     )
 
 
-def breakString(string, width=79, firstLineOffset=0):
+def breakString(str, width=79, firstLineOffset=0):
```

Comment:
Calling this `str` is also not good because it is a built-in Python type.

Reply:
oh...right!! Does `line` sound ok?

Reply:
since whatever it is may be broken into lines, maybe call it `text`?